app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,10 +1,10 @@
 class Admin::DashboardController < AdminController
 
   def index
-    @cohorts = Cohort.all.to_json(:include => [{:applications => {:include => [:user, :reviews]}}, :reviewers, :non_reviewers], methods: [:formatted_close_date, :formatted_start_date, :formatted_notify_date])
+    @cohorts = Cohort.all.to_json(:include => [{:applications => {:include => [:user, :reviews]}}, :reviewers, :non_reviewers, :order_by_id_and_state => {:include => [:user, :reviews]}], methods: [:formatted_close_date, :formatted_start_date, :formatted_notify_date])
```

Comment:
What does this change do? Are we not already including applications? What if we sorted client-side? This blob of data is getting crazy!

Reply:
Yeah, I agree but I was more going with the app to add the features for now before refactoring but it seems that a serializer would do something similar.